<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Bernd Stramm</author>
 <class>LocoMain</class>
 <widget class="QMainWindow" name="LocoMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loco</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../loco.qrc">
    <normaloff>:/loco.png</normaloff>:/loco.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextBrowser" name="msgLog"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="loco::LightMap" name="displayMap" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>498</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="destination">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Destination Unknown</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAgenda">
    <property name="title">
     <string>Loco</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help...</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuAgenda"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNewItem">
   <property name="text">
    <string>New Item</string>
   </property>
  </action>
  <action name="actionClearOld">
   <property name="text">
    <string>Clear Old</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionRevive">
   <property name="text">
    <string>Revive Expired</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>loco::LightMap</class>
   <extends>QWidget</extends>
   <header location="global">src/light-map.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../loco.qrc"/>
 </resources>
 <connections/>
</ui>
